api/service/syncing/syncing.go

```diff
@@ -158,9 +148,12 @@ func (peerConfig *SyncPeerConfig) GetBlocks(hashes [][]byte) ([][]byte, error) {
 }
 
 // CreateSyncConfig creates SyncConfig for StateSync object.
-func (ss *StateSync) CreateSyncConfig(peers []p2p.Peer) {
+func (ss *StateSync) CreateSyncConfig(peers []p2p.Peer) bool {
 	utils.GetLogInstance().Debug("CreateSyncConfig: len of peers", "len", len(peers))
-	utils.GetLogInstance().Debug("CreateSyncConfig: len of peers", "peers", peers)
+	if peers == nil || len(peers) == 0 {
```

Comment:
nit: peers == nil means len(peers) == 0 in golang

Reply:
so enoguh to check `if len(peers) == 0`

Reply:
changed it